packages/common/src/types/chainClients.ts

```diff
@@ -0,0 +1,65 @@
+import { RequireAtLeastOne } from "type-fest";
+import { BlockHeader } from "./block";
+import { Box } from "./boxes";
+import { HexString } from "./common";
+import { SignedTransaction, TransactionId, UnsignedTransaction } from "./transactions";
+
+export type QueryBase = {
+  take?: number;
+  skip?: number;
+};
+
+export type BoxQuery<W extends BoxWhere> = {
+  where: RequireAtLeastOne<W>;
+
+  /**
+   * Determines if should include unspent boxes from the mempool.
+   * @default true
+   */
+  includeUnconfirmed?: boolean;
```

Comment:
In the comment, it says that the default value is true, but the definition doesn't reflect that :thinking:

Reply:
Yes, this means that if `includeUnconfirmed` is undefined, then it must default to `true`.